.readthedocs.yaml

```diff
@@ -0,0 +1,9 @@
+vesion: 2
```

Comment:
```suggestion
version: 2
```
Typo?

Reply:
this explains why i couldn't get RTD to build with conda lololol :D